<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QMainWindow" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 7px;
	color: rgb(62, 231, 41);
	background-color: rgb(40, 109, 50);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>-140</x>
         <y>60</y>
         <width>661</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Scrabble&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="exit_button">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>10</y>
         <width>14</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 40, 54);
	border-radius: 7px;
	border: none
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="emoji_label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>340</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/images/3x.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QStackedWidget" name="login_reg_stacked">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>251</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: transparent;
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="login_content">
        <widget class="QFrame" name="login_container">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>10</y>
           <width>251</width>
           <height>171</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="error_label">
          <property name="geometry">
           <rect>
            <x>27</x>
            <y>100</y>
            <width>201</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: red;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="register_button">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>130</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color: rgb(206, 202, 207);
	background-color: rgb(188, 85, 49);
		border-radius:7px;
}QPushButton::pressed{
	background-color: rgb(174, 69, 77);
}</string>
          </property>
          <property name="text">
           <string>nowy gracz?</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="password_input">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>211</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 10px;
	color: rgb(206, 202, 207);
	background-color: rgb(55, 148, 67);
lineedit-password-character: 9679;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="login_button">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>130</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color: rgb(206, 202, 207);
		background-color: rgb(150, 38, 26);
		border-radius:7px;
}QPushButton::pressed{
	background-color: rgb(174, 69, 77);
}</string>
          </property>
          <property name="text">
           <string>zaloguj</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="login_input">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>211</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 10px;
	color: rgb(206, 202, 207);
	background-color: rgb(55, 148, 67);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>login</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>0</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;Zaloguj siÄ™&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="register_content">
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color: transparent;
}</string>
        </property>
        <widget class="QFrame" name="register_container">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>251</width>
           <height>181</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: transparent;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="error_register_label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: red;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="init_register_button">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>130</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color: rgb(206, 202, 207);
	background-color: rgb(188, 85, 49);
		border-radius:7px;
}QPushButton::pressed{
	background-color: rgb(174, 69, 77);
}</string>
          </property>
          <property name="text">
           <string>zarejetruj</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="reg_password_input">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>211</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 10px;
	color: rgb(206, 202, 207);
	background-color: rgb(55, 148, 67);
lineedit-password-character: 9679;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="init_login_button">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>130</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(211, 238, 231);
	background-color: rgb(119, 192, 255);
		border-radius:7px;
border: 2px;
}QPushButton::pressed{
	border-color: rgb(44, 88, 54);
	background-color: rgb(174, 69, 77);
}</string>
          </property>
          <property name="text">
           <string>logowanie</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="reg_login_input">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>211</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 10px;
	color: rgb(206, 202, 207);
	background-color: rgb(55, 148, 67);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>login</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>0</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;UtwÃ³rz konto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../src_qt.qrc"/>
 </resources>
 <connections/>
</ui>
